### PR TITLE
feat: add gio-config module with form field appearance

### PR DESCRIPTION
**Pull request by gcusnieux**
**Issue**

https://gravitee.atlassian.net/browse/APIM-456

---
#### 🧪&nbsp;&nbsp;Gravitee.io Automatic Prerelease
```
npm install @gravitee/ui-particles-angular@5.1.4-apply-form-field-appearance-54a6c3c
```
```
yarn add @gravitee/ui-particles-angular@5.1.4-apply-form-field-appearance-54a6c3c
```

---

### CODE REVIEW

**Review by loriepisicchio — APPROVED** [review]

**Review by ThibaudAV — COMMENTED** [review]
I think we need to add this module to each ui-p stories. i can help you to do that if you want ?

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-headers/gio-form-headers.component.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
       [formGroupName]="headerIndex"
     >
       <td class="gio-form-headers__table__header-row__td-key">
-        <mat-form-field class="gio-form-headers__table__header-row__td-key__field" [floatLabel]="'never'">
+        <mat-form-field class="gio-form-headers__table__header-row__td-key__field" [floatLabel]="'never'" appearance="standard">
```
Comment (ThibaudAV) [inline ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-headers/gio-form-headers.component.html:32]: maybe we can keep this apparence for this use case no ?

Comment (ThibaudAV) [reply ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-headers/gio-form-headers.component.html:32]: but 🤷‍♂️

**Review by ThibaudAV — APPROVED** [review]

**Pull request merged**